modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -56,8 +56,12 @@ DomainIntegralAction::validParams()
                                 "The last ring of elements for volume integral domain");
   params.addParam<std::vector<VariableName>>(
       "output_variable", "Variable values to be reported along the crack front");
-  params.addParam<bool>(
-      "convert_J_to_K", false, "Convert J-integral to stress intensity factor K.");
+  MooseEnum output_type("J K C", "J");
+  params.addParam<MooseEnum>("output_type",
```

Comment:
One more thing -- we should keep the old option as a deprecated parameter, at least long enough to fix up the Grizzly tests so we don't break it with this PR.

Reply:
Should work now.

Reply:
I think we don't even want `output_type` as a parameter here.